sql/moz-fx-data-shared-prod/firefox_ios_derived/app_store_funnel_v1/query.sql

```diff
@@ -21,7 +22,7 @@ downloads_data AS (
   FROM
     app_store.firefox_downloads_territory_source_type_report
   WHERE
-    `date` >= '2022-01-01'
+    `date` = DATE_SUB(@submission_date, INTERVAL 7 DAY)
```

Comment:
The common lag is 5 days and to be safe we use 7 days. What happens when data arrives with a lag of 8 or more days?

Reply:
We will see a drop in the numbers and we would have to re-run the ETL for the affected dates. Unfortunately, there is no good way to detect this. I did add a note to checks.sql that metric variance check could be added to help detect this (assuming there is a % difference we expect to see from day to day).